website/vcd.erb

```diff
@@ -722,6 +740,7 @@
             </li>
             <li<%= sidebar_current("docs-vcd-resource-network-pool") %>>
               <a href="/docs/providers/vcd/r/network_pool.html">vcd_network_pool</a>
+            </li>
```

Comment:
Should we also put

```suggestion
            </li>
            <li<%= sidebar_current("docs-vcd-resource-multisite-site-association") %>>
              <a href="/docs/providers/vcd/r/multisite_site_association.html">vcd_multisite_site_association</a>
            </li>
            <li<%= sidebar_current("docs-vcd-resource-multisite-org-association") %>>
              <a href="/docs/providers/vcd/r/multisite_org_association.html">vcd_multisite_org_association</a>
            </li>
```
Even though this file is not used...

Reply:
Yep. Updated